modules/Behaviour/behaviour_manage_edit.php

```diff
@@ -100,18 +97,42 @@
                 $form->addHiddenValue('address', "/modules/Behaviour/behaviour_manage_add.php");
                 $form->addRow()->addClass('hidden')->addHeading('Step 1', __('Step 1'));
 
+
+                //To show other students involved in the incident
+                if(!empty($values['gibbonMultiIncidentID'])) {
+
+                    $resultList = $container->get(BehaviourGateway::class)->selectMultipleStudentsOfOneIncident($values['gibbonMultiIncidentID']);
```

Comment:
Using result variables is part of the legacy style that we're refactoring out. In this case, a nice concise way to fetch the students would be: 
```suggestion
                    $students = $container->get(BehaviourGateway::class)->selectMultipleStudentsOfOneIncident($values['gibbonMultiIncidentID'])->fetchAll();
```
The result object internally handles checking for no results and will return an empty array from the fetchAll() method.